src/get_games.py

```diff
@@ -108,25 +107,27 @@ def download_games(name: str,
     pprint("Total games seen: " + str(len_total_games))
     print(initial_time, latest_time, delta_time)
     print(convert_ms_to_date(initial_time),
-          convert_ms_to_date(latest_time),
-          convert_ms_to_date(delta_time))
+          convert_ms_to_date(latest_time))
 
     games_list = list()
 
     time_30min = 30 * 60 * 1000
     increment = max([int(delta_time / 100), time_30min])
-    latest_time = initial_time + 2 * increment
+
     for time_index in range(initial_time,
                             latest_time,
                             increment):
-        print(convert_ms_to_date(time_index))
+        print(convert_ms_to_date(time_index).__str__() +
```

Comment:
1. never call ```var.__str__ ```, call ```str(val)```
2. print() already calls str() on its arguments
3. if you want to align text, it's probably better to use f-strings

Reply:
1) Fixed.
2) Fixed.
3) No alignment needed, just want to separate the values. Let's keep this simple for now.